react-js/react-part-3-component-hierarchies.md

```diff
@@ -248,164 +229,153 @@ class App extends React.Component {
-
-  <ModalBody>
-    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-    tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-    quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    consequat.
-  </ModalBody>
-
-  <ModalFooter>
-    <Button color="primary" onClick={this.toggle}>
-      Do Something
-    </Button>{' '}
-    <Button color="secondary" onClick={this.toggle}>
-      Cancel
-    </Button>
-  </ModalFooter>
-</Modal>
+ReactDOM.render(<App />, document.querySelector('#react'));
 ```
 
-[Run code](https://codepen.io/annu12340/pen/dyYvVWB?editors=1010)
+[Reactstrap Buttons CodePen](https://codepen.io/vegetabill/pen/QWKYPrY)
 
 ### Guided Practice
 
-#### Let's create a random Quote Generator using react. The application would display a random text from an array when a user clicks on 'new quote' button
+We'll create a random quote display using React. The application will display a random quote when a user clicks on button
 
-[View demo](https://u79vw.csb.app/)
-[View code](https://codesandbox.io/s/random-quote-machine-u79vw?file=/src/components/QuoteText.js)
+[Starter Code](https://codepen.io/vegetabill/pen/RwGvOVg?editors=1010)
 
-#### Create a json file with some quotes
+1. QuoteGenerator - the top-level app
+1. QuoteText - displays the quote text and speaker name
+1. QuoteButton - user clicks on it to pick a new random quote
 
-First let us create an array of random quotes and save it in a json file
+#### Test Data
+
+The full set is in the starter code link above.
 
 ```js
-[
-  {
-    quote: 'Life isn’t about getting and having, it’s about giving and being.'
-  },
+// Source: https://www.mentalfloss.com/article/53181/inspiring-quotes-10-influential-women-tech
+const QUOTES = [
   {
-    quote: 'Whatever the mind of man can conceive and believe, it can achieve.'
-  },
-  {
-    quote: 'Strive not to be a success, but rather to be of value.'
+    text:
+      'A ship in port is safe, but that is not what ships are for. Sail out to sea and do new things.',
+    speaker: 'Grace Hopper'
   }
+  //...
 ];
-```
 
-#### Create the components for the quote generator app
-
-Our app has 2 components:-
+class QuoteGenerator extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>Women in Tech Random Quotes</h2>
+        <div>Quote Text</div>
+        <button>Quote Button</button>
+      </div>
+    );
+  }
+}
 
-- QuoteBox.js
-- QuoteText.js
+ReactDOM.render(<QuoteGenerator />, document.querySelector('#react'));
+```
 
-1. QuoteText.js
-   This component takes in a text and wraps it in a h1 tags with an id "text" using an arrow function. It is then exported back
+#### Create the components for the quote generator app
 
-```js
-import React from 'react';
-const QuoteText = ({ text }) => <h1 id="text">{text}</h1>;
-export default QuoteText;
-```
+You will need to replace the placeholders above with actual `QuoteText` and `QuoteButton` components.
```

Comment:
Should there be a step reminding them to clone your starter pen and code along? Or are they just able to work in it directly (clearly I dont use codepen)

Reply:
Haha I don't use it much either. Let me try it and see!

Reply:
Looks like no. You can just start editing so I think we can just omit. Also by this point in the curriculum, they probably know more about how to use CodePen than me.